datasource/hcp-packer-image/data.go

```diff
@@ -124,11 +124,14 @@ func (d *Datasource) Execute() (cty.Value, error) {
 
 	output := DatasourceOutput{}
 
+	cloudAndRegions := map[string][]string{}
 	for _, build := range iteration.Builds {
+		cloudAndRegions[build.CloudProvider] = cloudAndRegions[build.CloudProvider] // force creation
```

Comment:
This is failing linting and I'm not seeing the purpose here. Why not just declare an empty list?

Reply:
You're right ! Removing that line:
```suggestion
```